package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-ui-react",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
We more or less use git flow and wouldn't want to bump the version here but in the hotfix or release branch

Reply:
@nmanu1 could this have been picked up by a release you did? I think a Github action made this change

Reply:
I think because this branch was based off of main instead of develop, this PR is now picking up the changes that were in main but haven't been merged into develop yet. this change comes from the hotfix made in #318

Reply:
Tristan you can probably just merge your dev/filtergroup-css branch with develop (you shouldn't need to do a rebase just a regular merge normally works)